google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java

```diff
@@ -874,71 +877,95 @@ public void testListTableDataWithOptions() {
     assertArrayEquals(TABLE_DATA.toArray(), Iterables.toArray(page.getValues(), List.class));
   }
 
-  @Test
-  public void testCreateQueryJob() {
```

Comment:
Note: these old tests are essentially change detectors. They don't really do much beyond calling `toPb` twice gives the same result.